libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,21 +7,21 @@
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
+    <button type="search" aria-label = "Search Books" mat-icon-button matSuffix>
```

Comment:
You could make this `type="submit"`, but there is no `type="search` for buttons (you could move that to the input though). 

Also, it looks like you don't have auto-formatting on. Update your editor to format on save to avoid formatting inconsistencies.